src/include/network/routines.rb

```diff
@@ -712,9 +712,14 @@ def SetLinkUp(dev_name)
       Run("ip link set #{dev_name} up")
     end
 
-    def SetLinkDown(dev_name)
-      log.info("Setting link down for interface #{dev_name}")
-      Run("ip link set #{dev_name} down")
+    def SetLinkDown(dev_name, delete = false)
```

Comment:
Only if you want to call ifdown with delete option, in this case, we want to call it when renaming an interface.